xls/netlist/netlist.cc

```diff
@@ -214,6 +214,28 @@ absl::StatusOr<const Module*> Netlist::GetModule(
       absl::StrFormat("Module %s not found in netlist.", module_name));
 }
 
+absl::StatusOr<const CellLibraryEntry*> Netlist::GetOrCreateLut4CellEntry(
+    int64 lut_mask) {
+  if (lut_mask > ((1 << 16) - 1)) {
```

Comment:
Probably want to check that the lut_mask is not negative either.

Reply:
done